dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/program/EnrollmentServiceTest.java

```diff
@@ -65,6 +66,11 @@
  */
 @Transactional
 class EnrollmentServiceTest extends PostgresIntegrationTestBase {
+  private static final UID ENROLLMENT_A_UID = UID.of(CodeGenerator.generateUid());
```

Comment:
are we only accessing these uids as strings in here? can't see it form the diff 🤔 if yes can we keep them as strings to get rid of the `getValue()` we do all the time in here